src/Scorecard.js

```diff
@@ -0,0 +1,36 @@
+function Scorecard() {
+  this.frames = [];
+  this.score = 0;
+};
+
+Scorecard.prototype.enter = function(roll1, roll2) {
+  
+    if(roll1 + roll2 > 10){
+      throw Error()
+    }
+ else{
+  this.frames.push({firstRoll: roll1, secondRoll: roll2})
+
+ }
+};
+
+Scorecard.prototype.calculate = function() {
```

Comment:
According to rubric, good practice to have one function that calculates and one method that returns the pre-calculated score (eg, calculateScore - does all the calculating - and score - that returns just the score)